src/directive/sort.directive.ts

```diff
@@ -61,10 +61,10 @@ export class JhiSortDirective {
 
     private resetClasses() {
         const allThIcons = this.element.querySelectorAll(this.sortIconSelector);
-        allThIcons.forEach((value) => {
-            value.classList.remove(this.sortAscIcon);
-            value.classList.remove(this.sortDescIcon);
-            value.classList.add(this.sortIcon);
-        });
+        for (let i = 0; i < allThIcons.length; i++) {
```

Comment:
Please add a comment saying normal for loop is used insted if forEach due to IE issue. Else I might change it back accidentally in the future